### PR TITLE
Version update 2016/09

### PR DESCRIPTION
**Pull request by ibm2431**
My last recapture of starting cities certainly helped the batch job, but one zone in each town still ended up being off and required another recapture to guarantee accuracy.

West Ronfaure, South Gustaberg, and West Sarutabaruta have been increasingly used by SE for events this past year, and should probably be watched like starting cities. I initially didn't believe such a large shift at first and had to go out to retail Ronfaure to confirm.

### CODE REVIEW

File: scripts/zones/Caedarva_Mire/TextIDs.lua [hunk 1]
```diff
@@ -22,5 +22,5 @@ MEMBER_TOO_FAR = 7468; -- One or more party members are too far away from the en
 NOTHING_OUT_OF_ORDINARY = 7377; --There is nothing out of the ordinary here.
         NOTHING_HAPPENS = 7445; -- Nothing happens...
 
- JAZARAATS_HEADSTONE = 7520; -- The name ir Jazaraat is engraved on the headstone... 
+ JAZARAATS_HEADSTONE = 7520; -- The name ir Jazaraatis engraved on the headstone...
```
Comment (TeoTwawki) [inline scripts/zones/Caedarva_Mire/TextIDs.lua]: This one looks off.

File: scripts/zones/East_Ronfaure/TextIDs.lua [hunk 2]
```diff
@@ -13,7 +13,7 @@ LOGGING_IS_POSSIBLE_HERE = 7482; -- Logging is possible here if you have
 -- General Dialog
      RAYOCHINDOT_DIALOG = 7406; -- If you are outmatched, run to the city as quickly as you can.
      CROTEILLARD_DIALOG = 7407; -- Sorry, no chatting while I'm on duty.
-      BLESSED_WATERSKIN = 7451; -- To get water, rade the waterskin you hold with the river. 
+      BLESSED_WATERSKIN = 7451; -- To get water, radethe waterskin you hold with the river.
```
Comment (takhlaq) [inline scripts/zones/East_Ronfaure/TextIDs.lua]: trade the

File: scripts/zones/Caedarva_Mire/TextIDs.lua (hunk 1 above)
Comment (ibm2431) [inline scripts/zones/Caedarva_Mire/TextIDs.lua]: From the filtered-dialog-table-079.xml it wrote:
```
<thing type="DialogTableEntry"><field name="index">7520</field><field name="text">The name ir Jazaraatis engraved on the headstone... </field></thing>
```

I think it has something to do with the curly quotes around the name ir Jazaraat. I noticed the batch job wouldn't pick up on another curly quote (trading the waterskin I think) until both the offending quote _and_ the space after it were removed. To be honest, I have no clue how the tool works to grab fragments when the string is full of offensive characters. I just give it whatever will stop the FIXMEs. :sweat_smile:

File: scripts/zones/East_Ronfaure/TextIDs.lua (hunk 2 above)
Comment (TeoTwawki) [inline scripts/zones/East_Ronfaure/TextIDs.lua]: @takhlaq its prolly special char related, making the T invisible to the batch tools sight

Comment (ibm2431) [inline scripts/zones/East_Ronfaure/TextIDs.lua]: That's what I initially changed it to, but the tool wouldn't pick up on it. Probably due to the curly quotes around trade. So I plopped in what it spit out in the filtered table, reran, and it latched onto it that time.

Text field from POLUtils:
```
To get water, “trade” the waterskin you hold with the river.≺Prompt≻
```

filtered-dialog-table-101.xml
```
<thing type="DialogTableEntry"><field name="index">7451</field><field name="text">To get water, radethe waterskin you hold with the river. </field></thing>
```

Comment (TeoTwawki) [inline scripts/zones/East_Ronfaure/TextIDs.lua]: So its in the dump that way..*"polutils limitations*" :laughing:

**Pull request merged**